schduler/clock.py

```diff
@@ -21,6 +25,11 @@ def timed_trigger_updatedb_sec():
     q.enqueue(trigger_migratedb)
 
 
+@sched.scheduled_job('cron', day_of_week='mon-sun', hour=12)
+def timed_trigger_updatedb_sec():
+    q.enqueue(trigger_migratedb)
```

Comment:
why are we adding a second trigger_migratedb here, I assume this was supposed to be  trigger_update_job_priority_table() ?